electron/js/window-manager.js

```diff
@@ -17,15 +17,14 @@
  *
  */
 
-
-const {BrowserWindow} = require('electron');
+const { BrowserWindow } = require('electron');
 
 let primaryWindowId;
 
+const _getPrimaryWindow = () =>
+  primaryWindowId ? BrowserWindow.fromId(primaryWindowId) : BrowserWindow.getAllWindows()[0];
```

Comment:
Let's not run prettier on things. This easily fits one line in github views.

Reply:
I accidentally formatted it :) yep..